operator/main.py

```diff
@@ -211,10 +212,13 @@ def validate_volume_request(obj):
                       " specified")
         return False
 
+    # should'nt this come under disperse volume?? Not replica??
     if subvol_bricks_count > 1:
+        # Why not range(0, 1): since / is always == 1 ??
         for i in range(0, int(len(bricks) / subvol_bricks_count)):
             decommissioned = ""
             for k in range(0, subvol_bricks_count):
+                # why not brick_idx = k since value is always same, and i always 0
```

Comment:
Example: Replica count = 3, Number of bricks = 6. Inner loop iterates 0, 1, 2 and 0, 1, 2 two times.

with below line the index will be 0, 1, 2, 3, 4, 5

Reply:
Thanks alot got it